index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css" type="text/css">
+  <title>Title</title>
+</head>
+
+<body>
+  <div class="maincontainer">
+
+    <div class="header">
+      <div class="header__logo"><p>Passion</p></div>
+      <div class="header__line"><img src="./img/header/line.png" alt=""></div>
+      <div class="header__title"><h2>passion a responsive email&nbsp;template.</h2></div> 
```

Comment:
зачем снова обертка?
```suggestion
      <h2 class="header__title">passion a responsive email&nbsp;template.</h2>
```